src/components/common/MoodIcon.jsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+const MoodIcon = ({src, alt, width, height}) => (
+        <Image
+            src={src}
+            alt={alt}
+            width={width}
+            height={height}
+        />
+);
+
+const MoodIconContainer = ({mood, width, height}) => {
+    const src = `/icons/${mood}.png`;
+    const alt = `${mood} icon`;
+
+    return (
+        <MoodIcon
+            src={src}
+            alt={alt}
+            width={width || 100}
```

Comment:
Once you have the default value, you can just pass width as props instead of width || 100.

Reply:
I deleted "|| 100" since I set the default values to the parameters.